README.md

```diff
@@ -6,7 +6,7 @@
 
 This is the simplest and probably the most robust way to add live reloading to your Rails app.
 
-Just add the gem and thats it, now you have a live reloading. No 3rd party dependencies.
+Just add the gem and thats it, now you have a live reloading. **No 3rd party dependencies**.
```

Comment:
I don't believe it was completely true before. Even less after adding "faye-websocket" as a dependency.

Reply:
Our reasoning behind this saying is that you don't need anything other than this gem to get started. Obviously we could have implemented functionality of dependencies such as `listen` gem, or `faye-websocket` as it is really small, in fact, the initial version was self written, but then I found this gem and decided to go with it.

Reply:
no 3rd party dependency is something like Redis, a separate process or even Chrome extensions, or anything like that.
Of course, you can always tell that Faye is a dependency, but if it's complied with "bundle install" it's not a problem at all. This gem works by just adding it to the Gemfile and not doing any other steps.

Reply:
I understand your reasoning. Not relying on redis / actioncable is the reason I chose this gem over hotwire-livereload. I also think that utilizing small gems like `listen` and `faye-websocket` for the job is the way to go.
Still, stating **No 3rd party dependencies** and having three times `add_dependency` in a .gemspec is a little far-fetched in my opinion. But this can be my misinterpretation of '3rd party dependency'.
Anyway, keep up the great job 👍

Reply:
also partially agree with you, but we just consider such dependencies as very minor, almost at the level that gem requires to have Ruby installed in your system to make it works

Thank you for your support and for doing this code review. We hope you can contribute to this gem or others that we have.

Reply:
In the end, we decided to change this text as it was a little controversial because of the different definitions of 3rd party dependencies.